.travis.yml

```diff
@@ -0,0 +1,41 @@
+dist: xenial
+
+language: node_js
+
+node_js:
+  - "8"
```

Comment:
probably need to start updating this to 12...

```suggestion
  - "12"
```

Reply:
rgr, updated